cypher-shell/src/main/java/org/neo4j/shell/commands/Param.java

```diff
@@ -59,25 +59,20 @@ public String getHelp() {
     @Override
     public void execute(@Nonnull final String argString) throws CommandException {
         Matcher alphanumericMatcher = argPattern.matcher(argString);
-
         if (!alphanumericMatcher.matches()) {
```

Comment:
could you flip this conditional? It reads better if you check for a positive match IMO